run_tribler.py

```diff
@@ -18,6 +18,9 @@
     # Set up logging
     setup_gui_logging()
 
+    # Check sqlite databases and LevelDb stores
+    check_datastores()
```

Comment:
I argue that this should be moved to the Tribler core since this won't work if the GUI and Core are running on separate machines (I always make that assumption for myself).

Reply:
At this point in line 22, neither the GUI nor the core has started yet so I thought it would be a good place to do some preliminary database checks here. Also, we can have more extensive checks when the core is starting. This will surely increase the startup time so some benchmarking would be necessary.

Reply:
We can still run these checks before the session has started when moving the code to the core.

Creating a benchmark for booting times is a good idea. We should discuss this at the beginning of the next year 👍